### PR TITLE
Updating dependencies in order to update deep-insights with #11166

### PR DESCRIPTION
**Pull request by xavijam**
Related with #11166 and deep-insights PR https://github.com/CartoDB/deep-insights.js/pull/506.

### CODE REVIEW

**Comment by javierarce** [comment]
👋  Is this ready to be merged? It's needed to deploy https://github.com/CartoDB/cartodb/issues/11166

**Comment by xavijam** [comment]
@javierarce we have to point to master in deep insights package and regenerate shrink wrap again

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cartodb-ui",
-  "version": "4.5.80",
```
Comment (xavijam) [inline package.json]: I'd change the package version :D

**Comment by xavijam** [comment]
Thanks man! :es:

File: package.json (hunk 1 above)
Comment (javierarce) [inline package.json]: Oh, thanks!

**Comment by xavijam** [comment]
I've restarted one travis tests, it failed with a frontend test, not sure if it is consistant or nor, let's see.

**Comment by xavijam** [comment]
Travis ones passed (:dancer:), restarting onpremises frontend tests.

**Comment by javierarce** [comment]
This is taking ages… should we keep waiting for onpremises?

**Comment by matallo** [comment]
actually it failed (`11:16:42 Finished: FAILURE`) but I'm unable to restart :(

**Comment by xavijam** [comment]
Restarted once more, if it doesn't change let's go with it.

**Comment by xavijam** [comment]
GREEN!!!

**Pull request merged**